### PR TITLE
refactor(proxy): use thiserror for routing error messages

### PR DESCRIPTION
**Pull request by geigerzaehler**
Use `thiserror` for routing error messages

### CODE REVIEW

File: proxy/api/src/http/error.rs [hunk 1]
```diff
@@ -2,23 +2,26 @@
...
 
 /// HTTP layer specific rejections.
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
 pub enum Routing {
     /// The currently active [`coco::user::User`] is missing.
+    #[error("Owner is missing")]
     MissingOwner,
     /// The keystore is sealed, context does not have a signer.
+    #[error("No session has been createcd yet")]
```
Comment (FintanH) [inline proxy/api/src/http/error.rs:19]:
```suggestion
    #[error("No session has been created yet")]
```

**Review by FintanH — CHANGES_REQUESTED** [review]
One small fix :)

**Review by xla — APPROVED** [review]
➕ 👔 🌺 🍥

**Review by FintanH — APPROVED** [review]
:kissing::ok_hand:

**Pull request merged**